doc/rst/technotes/module_include.rst

```diff
@@ -0,0 +1,85 @@
+.. _readme-module_include:
+
+Including Sub-Modules from Separate Files
+=========================================
+
+This document describes draft support for including a submodule from a
```

Comment:
Why draft support?

Reply:
It gives a warning with `--warn-unstable` and we're expecting it to change in some way. That's also why it's documented in a technote and not in the spec.

Reply:
Gotcha. It might be worth describing this here, i.e. the feature is considered unstable and can be changed in the future.